### ISSUE TITLE
[a11y]:

### ISSUE THREAD

**Issue by anirudhaBapat** [carbon-design-system/carbon#21239]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.78.0

### React version

17.0.2

### Automated testing tool and ruleset

WCAG Mapping: 3.3.2 Labels or Instruction

### Assistive technology

JAWS_Chrome

### Description

Text input placeholder text screen reader issues

1. The screen reader does not announce the placeholder text information

<img width="654" height="337" alt="Image" src="https://github.com/user-attachments/assets/364b2c57-379d-4510-9d55-01cd9a4255a5" />

2. Screen reader announces Placeholder text information twice

<img width="729" height="354" alt="Image" src="https://github.com/user-attachments/assets/08b3f6f1-f753-4246-9ef5-62e3350488b8" />

### WCAG 2.1 Violation

WCAG Mapping: 3.3.2 Labels or Instruction

### Reproduction/example

https://staging-wa.watson-orchestrate.ibm.com/build/manage/voice

### Steps to reproduce

Go to Add voice -> follow the steps until step 2 and see the issue in the input box. 

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by 2nikhiltom** [comment]
Hey @anirudhaBapat ,

> Reproduction/example
> https://staging-wa.watson-orchestrate.ibm.com/build/manage/voice

I am not able to login to URL, I verified the [Storybook](https://react.carbondesignsystem.com/?path=/story/components-textinput--default ) and it is working fine.
Could you re-create  this issues on [stackblitz](https://stackblitz.com/github/carbon-design-system/sandboxes/tree/main/react/latest/react-19-vite?file=src%2FApp.jsx&preset=node=)

**Comment by anirudhaBapat** [comment]
@2nikhiltom I saw the storybook version is latest, this one is on `1.78.0`

**Comment by tay1orjones** [comment]
Closing due to no reproduction
